src/components/HorizontalTab/index.tsx

```diff
@@ -56,7 +52,6 @@ const HorizontalTab = ({selectedTab, onSelectedTabChange, tabs = []}: Horizontal
       alignItems: 'center',
       justifyContent: 'center',
       height: 48,
-      borderBottomColor: colors.black,
       borderBottomWidth: 2,
       opacity: 1
     },
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `borderBottomColor` property has been removed from the style object. If this was intentional and the color is now being set elsewhere, then this change is fine. However, if this was unintentional, you may want to add it back to ensure the border bottom color is set correctly.

```diff
- borderBottomWidth: 2,
+ borderBottomColor: colors.black,
+ borderBottomWidth: 2,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->